### PR TITLE
Add generated source

### PR DESCRIPTION
**Pull request by runejo**

### CODE REVIEW

**Comment by likp** [comment]
@runejo should we commit the generted files in the github workflow nuget.xml so that we make sure that we have the same source files commited as the once we build the nuget package with?

**Comment by runejo** [comment]
> @runejo should we commit the generted files in the github workflow nuget.xml so that we make sure that we have the same source files commited as the once we build the nuget package with?

Usually it is not recommended to let an Action write to the main branch. We have several other options
- change `nuget.yml` to **not** run openapi-converter and just build the soruce on main
- make a build action that fails if openapi spec and source code is not in sync (it should also do other validations and build tests)

**Comment by likp** [comment]
> > @runejo should we commit the generted files in the github workflow nuget.xml so that we make sure that we have the same source files commited as the once we build the nuget package with?
> 
> Usually it is not recommended to let an Action write to the main branch. We have several other options
> 
> * change `nuget.yml` to **not** run openapi-converter and just build the soruce on main
> * make a build action that fails if openapi spec and source code is not in sync (it should also do other validations and build tests)

We should probably have the later one. Any ideas on how to detect if they are out of sync?

**Comment by runejo** [comment]
yes, but I didn't have time to test it yesterday :-)

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        with:
+          generator: aspnetcore
+          openapi-file: PxAPI-2.yml
+          config-file: aspnetcore-generator-config.yml
+          generator-tag: v6.5.0
+          command-args: -o .
+
+      - name: Check for staged changes
+        run: |
+          git diff --exit-code
```
Comment (runejo) [inline .github/workflows/build.yml:38]: @likp this will detect if spec and code is out outof sync

Comment (likp) [reply .github/workflows/build.yml:38]: Nice. Should we removed the regeneration from the nuget workflow and just use the files already generated?

Comment (runejo) [reply .github/workflows/build.yml:38]: Yes, I'll do it first thing tomorrow😀

Comment (runejo) [reply .github/workflows/build.yml:33]: Ok, now it's done 👍

**Pull request merged**